src/app/centers/create-center/create-center.component.html

```diff
@@ -0,0 +1,129 @@
+<div class="container">
+
+  <mat-card>
+
+    <form [formGroup]="centerForm">
+
+      <mat-card-content>
+
+        <div fxLayout="column">
+
+          <mat-form-field>
+            <mat-label>Name</mat-label>
+            <input matInput required formControlName="name">
+            <mat-error *ngIf="centerForm.controls.name.hasError('required')">
+              Center Name is <strong>required</strong>
+            </mat-error>
+            <mat-error *ngIf="centerForm.controls.name.hasError('pattern')">
+              Center Name <strong>cannot</strong> begin with a special character or number
+            </mat-error>
+          </mat-form-field>
+
+          <mat-form-field>
+            <mat-label>Office</mat-label>
+            <mat-select required formControlName="officeId">
+              <mat-option *ngFor="let office of officeData" [value]="office.id">
+                {{ office.name }}
+              </mat-option>
+            </mat-select>
+            <mat-error *ngIf="centerForm.controls.officeId.hasError('required')">
+              Office is <strong>required</strong>
+            </mat-error>
+          </mat-form-field>
+
+          <mat-form-field>
+            <mat-label>Staff</mat-label>
+            <mat-select formControlName="staffId">
+              <mat-option *ngFor="let staff of staffData" [value]="staff.id">
+                {{ staff.displayName }}
```

Comment:
You can deactivate the field if staffData.length == 0. That will be more precise and user interactive.